modules/qtwidgets/include/modules/qtwidgets/properties/ordinalpropertywidgetqt.h

```diff
@@ -260,7 +260,7 @@ void OrdinalLikePropertyWidgetQt<Prop, Sem>::updateFromProperty() {
         val = util::spherical(val);
         min = T{std::numeric_limits<BT>::epsilon(), 0, -std::numbers::pi};
         max = T{3 * glm::length(max), std::numbers::pi, std::numbers::pi};
-        inc = T{glm::length(inc), std::numbers::pi / 100.0, 2 * std::numbers::pi / 100.0};
+        inc = T{inc.x, std::numbers::pi / 360.0, std::numbers::pi / 360.0};
```

Comment:
Should be 1 degree?
```suggestion
        inc = T{inc.x, std::numbers::pi / 180.0, std::numbers::pi / 180.0};
```

Reply:
Well it is quite arbitrary, and the main goal here is to get the number decimals to be 2, so I need to divide by something larger then pi * 100, hence 360, which will print something lie 3.14 instead of 3.1 in the text boxes.

Reply:
I don't follow. Why would 3.14... / 360 give you two decimals? 
This is just for the increment, which is not displayed?

Reply:
The increment is used to determine the number of shown decimals. See changes in [qtwidgets/inviwoqtutils.h](https://github.com/inviwo/inviwo/pull/1611/files#diff-8d49a3bcc050032d4a84dcfa4d5f1365e7d00b9146308f1ac38625f4f4ef4bbc)